connectors/flageval-connector.py

```diff
@@ -0,0 +1,129 @@
+import logging, json
+
+import aiohttp
+from aiohttp import ClientResponse
+from typing import Callable
+from moonshot.src.connectors.connector import Connector, perform_retry
+from moonshot.src.connectors.connector_prompt_arguments import ConnectorPromptArguments
+from moonshot.src.connectors_endpoints.connector_endpoint_arguments import (
+    ConnectorEndpointArguments,
+)
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FlagJudgeConnector(Connector):
+    def __init__(self, ep_arguments: ConnectorEndpointArguments):
+        # Initialize super class
+        super().__init__(ep_arguments)
+
+    @Connector.rate_limited
+    @perform_retry
+    async def get_response(self, prompt: str, prediction: str, ground_truth: str) -> str:
```

Comment:
noticed that there are 2 additional positional fields, prediction and ground_truth. 
as our base doesnt have this additional field, it will fail

Reply:
Actually, I have passed the recipe test!  :) Does passing the recipe test signify success?

Reply:
Hmm interesting. You used the cli?

Reply:
Yes, just as the doc mentioned. By the way, is there any issue here? The only way to call this function is through get_prediction, and I've modified it so that we can call it successfully.

Reply:
Nope! everything is good. 
Previously was using the flageval-flagjudge as the endpoint and hence faced an issue.
After seeing through the code in greater detail, then realise that flageval-flagjudge will be used by the metric for querying.